pyensembl/download_cache.py

```diff
@@ -111,8 +112,12 @@ def __init__(
                 annotation_name=annotation_name,
                 annotation_version=annotation_version)
 
-            self._cache_directory_path = datacache.get_data_dir(
-                subdir=self.cache_subdirectory)
+            if environ.get(ENV_KEY) is None:
```

Comment:
Correct — but I wanted to make this more explicit as it is not apparent from the function call what will happen if you pass both variables.

Actually, maybe adding a comment there to explain this might be better. Let me do that :+1: